apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -338,7 +345,32 @@ def render(element_html, data):
         )
 
         mcq_options = data["params"][answer_name]
-        mcq_options = filter_multiple_from_array(mcq_options, ["inner_html", "uuid"])
+        mcq_options = filter_multiple_from_array(mcq_options, ["inner_html", "uuid", "tag", "distractor-for"])
+
+        # visual pairing
+        correct_tags = set(block['tag'] for block in mcq_options + student_previous_submission if block['tag'] is not None)
```

Comment:
You use `mcq_options + student_previous_submission` in multiple places, would be best to set  this as a variable and re-use.

Reply:
It looks like `student_previous_submission` is actually an empty list some of the time and so its not even needed. I'm going to refactor a little bit.